Projects/Core/Networking/Model/Sources/Auth/OAuthResponse.swift

```diff
@@ -10,12 +10,12 @@ import Foundation
 import FirebaseAuth
 
 public struct OAuthResponseModel : Equatable {
-  var accessToken: String
-  var refreshToken: String
-  var credential: AuthCredential?
-  var email: String
-  var uid: String
-  
+  public var accessToken: String
+  public var refreshToken: String
+  public var credential: AuthCredential?
+  public var email: String
+  public var uid: String
```

Comment:
```suggestion
  public let accessToken: String
  public let refreshToken: String
  public let credential: AuthCredential?
  public let email: String
  public let uid: String
```

인스턴스 생성 후 변경하는 코드가 없는 것 같은데, immutable하게 let으로 바꾸는건 어떨까요?!

Reply:
넵!!!